ui/raidboss/data/06-ew/trial/golbez-ex.ts

```diff
@@ -382,6 +390,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: '(in + spread, for later)',
           de: '(rein + verteilen, für später)',
+          fr: '(intérieur + écartement, pour après',
```

Comment:
Might be missing the close parenthesis? ")"

Reply:
Good catch 👍